### ISSUE TITLE
Monomerization mode requires better handling of Ligands in own Chain

### ISSUE THREAD

**Issue by TJGorrie** [xchem/fragalysis-api#49]
Current behaviour of -m mode on xcimporter splits the pdb file according to chain using __pymol__. This has two consequences:

1) We need to split the pdb using gemmi.

2) If a ligand is in its own chain: the chain will be seperated into its own pdb file. This means that when we attempt to align the monomerized files. The ligand file is NOT aligned and the other ligand-less chains are lost (as they are not interesting). So it is not behaving as expected...

A solution may be to assign the Ligand C to the nearest chain using some magic (not sure). There is probably a quick win to do this using gemmi...

To summarise:
1) Ensure ligand is associated with the nearest protein chain
2) Split pdb file using gemmi.
3) Cry
